### PR TITLE
mention the sequence operator

### PR DESCRIPTION
**Pull request by stoned**
while here document the various way to build a Seq at the
start of the document.

### CODE REVIEW

File: doc/Type/Seq.pod6 [hunk 1]
```diff
@@ -11,6 +11,19 @@ C<Seq> is born in a state where iterating it will consume the values.
...
+A high-level construct to generate a C<Seq> is L«C<gather/take>|/syntax/gather%20take»,
+as well as many built-in methods like L«C<map>|/routine/map» and
+L«C<grep>|/routine/grep», low-level constructors to create a
+C<Seq> from an iterator or from looping constructs are available too.
+
+A C<Seq> can also be constructed with the
+L«sequence operator C<...>|/language/operators#infix_...» or one of its
+variants.
+
+    my $s = (1...5);
```
Comment (coke) [inline doc/Type/Seq.pod6:23]: the parens are not needed here, right? can we remove them?

File: doc/Type/Seq.pod6 [hunk 2]
```diff
@@ -11,6 +11,19 @@ C<Seq> is born in a state where iterating it will consume the values.
...
+    say $s;              # OUTPUT: «1 2 3 4 5)␤»
```
Comment (coke) [inline doc/Type/Seq.pod6:24]: Mismatched )

**Review by coke — COMMENTED** [review]
In general, fine, some minor nits on the example code.

File: doc/Type/Seq.pod6 (hunk 2 above)
Comment (stoned) [reply doc/Type/Seq.pod6]: indeed! Fixed!

File: doc/Type/Seq.pod6 (hunk 1 above)
Comment (stoned) [reply doc/Type/Seq.pod6:23]: AFAIU to assign a Seq to a scalar it is needed. Or else only the LHS of the sequence operator is assigned to the scalar.

``` 
$ cat /tmp/e.raku 
my $s = 1 ... 5;
($s.^name, $s).say;
$ raku /tmp/e.raku
Potential difficulties:
    Useless use of ... in sink context
    at /tmp/e.raku:1
    ------> my $s = 1 ...⏏ 5;
(Int 1)
```

**Review by JJ — APPROVED** [review]
Thanks!

**Pull request merged**